### DISCUSSION TITLE
How to install locale-gen in Docker container

### DISCUSSION THREAD

**Discussion by cbounds89** [firefly-iii/firefly-iii#5745] [Firefly III]
On how to change the locale so fireflyiii will read decimals correctly.  Currently 1 = 100

Fireflyiii was installed using docker-compose on a Linux machine. 

https://docs.firefly-iii.org/firefly-iii/advanced-installation/locales/

I tried following the instructions here
https://docs.firefly-iii.org/firefly-iii/faq/self_hosted/

But when I’m in the docker container “locale-gen” wouldn’t work cause it’s not recognized.

Thank you in advance.

**Comment by JC5** [comment]
Try this:
 ```
apt install locales
locale-gen
```

**Reply by cbounds89** [reply]
Thank you for the reply but I get “bash: apt: command not found” when entering the commands

**Reply by cbounds89** [reply]
Umm I see the issue now. 
Didnt realize I was using a custom image.

**Reply by JC5** [reply]
Which one were you using?

**Reply by cbounds89** [reply]
https://github.com/fabiodcorreia/docker-fireflyiii

I was trying to find a docker compose file with the CSV importer. I’ll move over to this image. Thank you for your help.

**Reply by JC5** [reply]
https://github.com/firefly-iii/docker/blob/main/docker-compose-data.yml

**Reply by cbounds89** [reply]
Thanks!
